### ISSUE TITLE
Still doesnt work with 7.10

### ISSUE THREAD

**Issue by GamesAndCheese** [espositos/fvtt-tokenactionhud#263]
I have tried everything under the sun. Just installed the newest update that was pushed out (1.0.31) which wont work. I keep receiving this error. I was told to try and revert back to the previous version. Any help would be cool
![image](https://user-images.githubusercontent.com/85651975/121439847-5bd03000-c94c-11eb-93d6-78c3f81c9150.png)

**Comment by Silzin** [comment]
I donnt code, so I don't understand the code you added here.  but I think I was having a similar problem.  what I did was go to the main page and download an earlier version.  I was recommended to get the 1.0.19 .  https://github.com/espositos/fvtt-tokenactionhud/releases/tag/1.0.19  after that all I did was copy the contents of the Zip into the Module folder for this module on my computer.

**Comment by espositos** [comment]
Yes, I'm sorry, until I have time to look into it, please downgrade to 1.0.19
